src/rvs_blas.cpp

```diff
@@ -218,6 +218,20 @@ bool rvs_blas::init_gpu_device(void) {
     return false;
   }
 
+  if("hiprand" == matrix_init) {
+
+    // Create hipRAND generator, assign stream.
+    if(hiprandCreateGenerator(&hiprand_generator, HIPRAND_RNG_PSEUDO_DEFAULT) != HIPRAND_STATUS_SUCCESS) {
+      std::cout << "\n hiprandCreateGenerator() failed !!!" << "\n";
+      return false;
```

Comment:
all returns without cleaning up explicitly allocated values like stream or blas allocate will result in resource leakages. Maybe add as a TODO but error returns are not used to do any cleanup even in caller heirarchy

Reply:
hiprandDestroyGenerator() called in rvs_blas::release_gpu_matrix_mem() to release rand generator alongside rocblas handle and hip stream.